vector/src/main/res/layout/view_read_receipts.xml

```diff
@@ -16,6 +16,8 @@
         android:paddingStart="4dp"
         android:paddingEnd="4dp"
         android:textColor="?vctr_content_primary"
+        android:textSize="12dp"
```

Comment:
to double check, the container views in this area look like they're `wrap_content` and the drawable has no hardcoded heights which means removing the text view's [fixed height](https://github.com/vector-im/element-android/pull/5856/files#diff-b4585bb9d47cfa459cbc36c3397fcfbfe131ce5e790ec7865baaf8d24d89b68dR12) might be enough :crossed_fingers: 

it has the benefit of allowing us to remain friendly to large text size users

Reply:
@ouchadam Just to double-check, you'd like the TextView to be taller than the avatar ImageViews? (I'm fine with it)

Reply:
Yep! Ideally the profile pictures would also scale with the text size by avoiding the hardcoded sizes and constraining to the height of the `TextView` itself but that could be another ticket/second iteration.

For the original issue symptom of clipped text I would lean towards it being better to have the text readable for a user with large fonts enabled at the cost of some UI imperfections, but not a strong opinion since this particular text isn't critical to the page

Reply:
My 2 cents: I agree with the original fix, since having a big font here is maybe not necessary for a11y.

Maybe a better approach could be to enlarge the Read Receipt avatar too, which is probably a better solution. We can have avatar size in `sp` to do that.